src/main/java/no/nav/veilarbaktivitet/service/CroneService.java

```diff
@@ -27,10 +27,11 @@ public void sendMoteServicemelding() {
         }
     }
 
-    @Scheduled(fixedRate = 1000, initialDelay = 1000)
+    @Scheduled(fixedRate = 500, initialDelay = 5000)
     public void sendMeldingerPaaKafka() {
         if (leaderElectionClient.isLeader()) {
             aktiviteterTilKafkaService.sendOppTil5000AktiviterPaaKafka();
+            aktiviteterTilKafkaService.sendOppTil5000AktiviterPaaKafkaV4();
```

Comment:
Forstår ikke helt hvorfor det er to men er det at det skal det legges på to køer midlertidig?

Reply:
stemmer v3 skal slettes når v4 har tatt over

Reply:
Burde vel kanskje kjøre i hver sin cron jobb?

Reply:
Stemmer, skal fjerne V3 etter at vi har startet å lytte til V4 i prod. Er redd for at det er 1 sekund, helt på starten, med meldinger som ikke blir sendt hvis vi ikke sender på V3 og V4 samtidig, og i tillegg så er ikke vi helt klare for V4 enda

Reply:
Ok, trenger ikke endre det for min del